src/test/java/io/lettuce/core/cluster/ClusterSetup.java

```diff
@@ -38,7 +38,7 @@
      *
      * @param clusterRule
      */
-    public static void setup2Masters(ClusterRule clusterRule) {
+    public static void setup2Masters(ClusterTestHelper clusterRule) {
```

Comment:
should we also rename `clusterRule ` to `clusterTestHelper`?(maybe)

Reply:
Absolutely yes, fixed.